### PR TITLE
init project

### PR DESCRIPTION
**Pull request by hoanghuong**
https://edu-redmine.framgia.vn/issues/232

### CODE REVIEW

**Comment by hoanghuong** [comment]
Init project : Create base project
Ready

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <activity android:name=".MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+    </application>
+
+</manifest>
```
Comment (vitconpc) [inline app/src/main/AndroidManifest.xml]: enter last line

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+</android.support.constraint.ConstraintLayout>
```
Comment (vitconpc) [inline app/src/main/res/layout/activity_main.xml]: enter last line

**Comment by hoanghuong** [comment]
sửa lỗi hash code
Ready

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@drawable/ic_launcher_background" />
+    <foreground android:drawable="@drawable/ic_launcher_foreground" />
+</adaptive-icon>
```
Comment (vitconpc) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml]: enter last line

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@drawable/ic_launcher_background" />
+    <foreground android:drawable="@drawable/ic_launcher_foreground" />
+</adaptive-icon>
```
Comment (vitconpc) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml]: enter last line

**Comment by hoanghuong** [comment]
ready

**Comment by DoanVanToan** [comment]
merged

**Pull request merged**